extra_data/cli/make_virtual_cxi.py

```diff
@@ -35,8 +51,15 @@ def main(argv=None):
         '--min-modules', type=int, default=9, metavar='N',
         help="Include trains where at least N modules have data (default 9)"
     )
+    ap.add_argument(
+        '-v', '--fill-value', action='append', nargs=2, metavar=('DS', 'V'),
```

Comment:
I'd prefer to skip the `-v` short form. An option taking two arguments is a bit fiddly for a short option anyway, and `-v` commonly means 'verbose mode', which it's conceivable we'd want some day.